sqs/src/main/scala/akka/stream/alpakka/sqs/SqsAckSettings.scala

```diff
@@ -14,19 +16,39 @@ final case class SqsAckSinkSettings(maxInFlight: Int) {
 }
 //#SqsAckSinkSettings
 
+object SqsBatchAckFlowSettings {
+  val Defaults = SqsBatchAckFlowSettings(
+    maxBatchSize = 10,
+    maxBatchWait = 500.millis,
+    concurrentRequests = 1
+  )
+}
+
+//#SqsBatchAckFlowSettings
+final case class SqsBatchAckFlowSettings(maxBatchSize: Int, maxBatchWait: FiniteDuration, concurrentRequests: Int) {
```

Comment:
Could you add Java-friendly, `withMaxBatchSize`, `withMaxBatchWait(Long, TimeUnit)` and `withConcurrentRequests`, please.